### PR TITLE
Add arabic translations

### PR DESCRIPTION
**Pull request by Abeerm171**
some countries are translated in Arabic language as you want , and as an additional option I added another translation in Persian language

### CODE REVIEW

**Comment by Abeerm171** [comment]
This pull request for issue #220 ✅

File: countries.json [hunk 1]
```diff
@@ -41,7 +41,9 @@
 			"slk": {"official": "Aruba", "common": "Aruba"},
 			"spa": {"official": "Aruba", "common": "Aruba"},
 			"fin": {"official": "Aruba", "common": "Aruba"},
-			"zho": {"official": "\u963F\u9C81\u5DF4", "common": "\u963F\u9C81\u5DF4"}
+			"zho": {"official": "\u963F\u9C81\u5DF4", "common": "\u963F\u9C81\u5DF4"},
```
Comment (herrjemand) [inline countries.json]: They need to look like these

**Comment by mledoze** [comment]
@Abeerm171 any updates?

**Comment by mledoze** [comment]
@Abeerm171 I am sorry but I will have to close this pull request if it doesn't receive any updates. Do you think you can finish it?

Thank you for your help

**Pull request closed without merging**